Solution1/DataAccessLayer/Models/HistoryWeatherRequest.cs

```diff
@@ -0,0 +1,13 @@
+﻿using System;
+
+namespace DataAccessLayer.Models
```

Comment:
Data models are only models which are saved in db. Is it saved? db knows nothing about any requests